### ISSUE TITLE
Wrong Path in recalboxFiles.py

### ISSUE THREAD

**Issue by WulfmanGer** [Voljega/amiga4recalbox#13]
https://github.com/Voljega/amiga4recalbox/blob/master/installAmiga/configgen/recalboxFiles.py Line: 31

, 'amiberry' : '/recalbox/share/emulateurs/amiberry'

correct: 
, 'amiberry' : '/recalbox/share/emulateurs/**amiga**/amiberry'

Interestingly, nothing happens with the wrong path. Everything's fine. In my opinion, this should be corrected anyway - perhaps this is important in recalbox/configgen later on?

**Comment by Voljega** [comment]
yes you're right, this is a mistake on my part but that path isn't really used right now, I'll correct it when putting the real path at integration time and using them in the code

**Comment by Voljega** [comment]
corrected in official version
